sdk/program/src/stake/config.rs

```diff
@@ -1,10 +1,8 @@
 //! config for staking
 //!  carries variables that the stake program cares about
 
-use {
-    super::state::{DEFAULT_SLASH_PENALTY, DEFAULT_WARMUP_COOLDOWN_RATE},
-    serde_derive::{Deserialize, Serialize},
-};
+pub use super::state::{DEFAULT_SLASH_PENALTY, DEFAULT_WARMUP_COOLDOWN_RATE};
```

Comment:
```suggestion
#[deprecated(
    since = "1.16.7",
    note = "Please use `solana_sdk::stake::state::{DEFAULT_SLASH_PENALTY, DEFAULT_WARMUP_COOLDOWN_RATE}` instead"
)]
pub use super::state::{DEFAULT_SLASH_PENALTY, DEFAULT_WARMUP_COOLDOWN_RATE};
```